### PR TITLE
 Delete IDE releated files in .gitignore

### PR DESCRIPTION
**Pull request by WoohyungHan**
# Describe what this PR does #
    The IDE is a user preference, and they should configure in
    individual envrionment. Delete IDE releated files in the
    gitignore.

## Is there anything that requires special attention ##
NO

## Related issues ##
NO

## Future concerns ##
NO

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,6 @@ Session.vim
 *.orig
 *.patch
 *.rej
+
+# Idea(goland) files
+.idea/
```
Comment (Madhu-1) [inline .gitignore]: can you add `.vscode/` also?

Comment (WoohyungHan) [reply .gitignore]: Done!

File: .gitignore [hunk 2]
```diff
@@ -20,3 +20,7 @@ Session.vim
 *.orig
 *.patch
 *.rej
+
+# goland, vscode
```
Comment (Madhu-1) [inline .gitignore]: can you please update the comment like `IDE related files (golang,vscode)`

Comment (WoohyungHan) [reply .gitignore]: oh, sorry about detail... done!

**Comment by Madhu-1** [comment]
> I would prefer users to add these files to their `$HOME/git/ignore` or similar personal configuration. See https://git-scm.com/docs/gitignore for details.

in that case, we need to remove `emac and vim related files from gitignore`

**Comment by WoohyungHan** [comment]
> > I would prefer users to add these files to their `$HOME/git/ignore` or similar personal configuration. See https://git-scm.com/docs/gitignore for details.
> 
> in that case, we need to remove `emac and vim related files from gitignore`

It looks better to me.

**Comment by nixpanic** [comment]
On Wed, Oct 16, 2019 at 03:46:04AM -0700, Madhu Rajanna wrote:
> > I would prefer users to add these files to their `$HOME/git/ignore` or similar personal configuration. See https://git-scm.com/docs/gitignore for details.
> 
> in that case, we need to remove `emac and vim related files from gitignore`

Yes, I agree. I do not think it makes sense to add directories and files
for all possible IDEs. The IDE is a user preference, and they should
configure their git environment according to their preferences.

**Comment by WoohyungHan** [comment]
> > I would prefer users to add these files to their `$HOME/git/ignore` or similar personal configuration. See https://git-scm.com/docs/gitignore for details.
> 
> in that case, we need to remove `emac and vim related files from gitignore`

Could I delete it or not?

**Comment by Madhu-1** [comment]
> > > I would prefer users to add these files to their `$HOME/git/ignore` or similar personal configuration. See https://git-scm.com/docs/gitignore for details.
> > 
> > 
> > in that case, we need to remove `emac and vim related files from gitignore`
> 
> Could I delete it or not?

please delete and update the PR and PR description and commit message

**Pull request merged**